dbus-org.sailfishos.usermanager.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Sailfish login service
+
+[Service]
+Type=dbus
+BusName=org.sailfishos.usermanager
+ExecStart=/usr/bin/sailfish-logind
```

Comment:
I got an error with this:
```
Mar 10 18:24:35 Sailfish systemd[1]: Starting Sailfish login service...
Mar 10 18:24:35 Sailfish systemd[7045]: dbus-org.sailfishos.usermanager.service: Failed at step EXEC spawning /usr/bin/sailfish-logind: No such file or directory
Mar 10 18:24:35 Sailfish systemd[1]: dbus-org.sailfishos.usermanager.service: Main process exited, code=exited, status=203/EXEC
Mar 10 18:24:35 Sailfish systemd[1]: Failed to start Sailfish login service.
Mar 10 18:24:35 Sailfish systemd[1]: dbus-org.sailfishos.usermanager.service: Unit entered failed state.
Mar 10 18:24:35 Sailfish systemd[1]: dbus-org.sailfishos.usermanager.service: Failed with result 'exit-code'.
```
Seems to be wrong binary name.

Reply:
Fixed, along with some cleanups.